### PR TITLE
Updated documentation for new installs and tests

### PR DESCRIPTION
**Pull request by tanaes**
Just updating the documentation to be a bit more clear.

### CODE REVIEW

**Comment by tanaes** [comment]
Hey @tkosciol @mortonjt @qiyunzhu mind taking a quick look at this for sanity?

File: README.md [hunk 1]
```diff
@@ -6,96 +6,136 @@
...
-To run actual tools on a simple set of test data, run:
+You can install all of these modules by executing:
 
 ```bash
-source activate oecophylla
+oecophylla install all
+```
 
-oecophylla workflow --test
+Fair warning: installing all modules will take a  bit of time.
```
Comment (qiyunzhu) [inline README.md:52]: extra whitespace

File: README.md [hunk 2]
```diff
@@ -6,96 +6,136 @@
...
 
-oecophylla workflow --test MODULE
+```bash
+oecophylla workflow --test -o test_out qc taxonomy
 ```
 
-## Philosophy
+## Basic tutorial
+
+For this tutorial, we'll analyze a pair of shallowly sequenced  microbiome samples distributed with the repository. They are located in `oecophylla/test_data/test_reads`.
```
Comment (qiyunzhu) [inline README.md:80]: extra whitespace

File: README.md [hunk 3]
```diff
@@ -6,96 +6,136 @@
...
-##### D. (cont.)
-* ["bin the bins" tool to be determined]
+These modules each will list per-sample outputs---for example, trimmed reads or assembled contigs---in per-sample directories within the module directory. 
 
-##### E. Downstream genome annotation
-* [Prokka]() - gene calling and annotation
-* []() - metabolic reconstruction
-* []() - molecule prediction
--->
+Combined outputs---for example, the MultiQC summaries or combined biom table taxonomic summaries---will be found in their own dictories within the primary module directory.
```
Comment (qiyunzhu) [inline README.md:141]: dictories => directory

File: README.md [hunk 4]
```diff
@@ -6,96 +6,136 @@
...
-To speed development, Travis is currently only testing the validity of the module installs and checking the snakefiles syntax, by using the `--dryrun` option in Snakemake:
+source activate oecophylla
 
+pip install -e .
 ```
-snakemake all --cores 2 --configfile config.yaml --dryrun
+
+### Installing modules
+
+The code to execute module commands is all located in the Oecophylla repository. The actual tools called by these modules, however, must be installed separately in their own environemnts.
```
Comment (qiyunzhu) [inline README.md:38]: environemnts => environments

File: README.md [hunk 5]
```diff
@@ -6,96 +6,136 @@
...
 
-The software tools carrying out each of the steps are as follows:
+The *environment file* simply lists the commands necessary to invoke the correct environment for each module. The included example file specifies the commands necessary for the Oecophylla-installed Conda environments, but these can be changed if, for example, you have appropriate environments specified as modules on a cluster.
 
-#### I. Quality control
+The *parameters file* specifies the parameters for each tool being executed, including paths to the relevant databases.
 
-* [MultiQC](http://multiqc.info) - aggregates QC results from several tools
-* [Atropos](https://github.com/jdidion/atropos) - read trimming and filtering
+Information from these three files is combined by Oecophylla into a single `config.yaml` configuration file and placed in the output directory of a run. This serves as complete record of the settings chosen for execution of a run, as well as instructions for restarting or extending the processing on a dataset for subsequent invokations of Oecophylla.
```
Comment (qiyunzhu) [inline README.md:98]: invokations => invocations

**Review by qiyunzhu — APPROVED** [review]
Hi @tanaes Sorry for the late reply. I think this is very good. A few minor typos. I also think that a brief "quick start" in the beginning of the document will be good, because the full-size doc may be too long for the impatient users. Just thoughts. Don't have to change at this moment.

**Pull request merged**